rental.rb

```diff
@@ -0,0 +1,21 @@
+require 'date'
+
+class Rental
+  attr_accessor :date
+
+  def initialize(book, owner, date = DateTime.now())
+    @date = date
+    self.book = book
+    self.owner = owner
```

Comment:
- [ ] the rental should add itself to the @book and the @owner achieving the 'belongs-to' association, that will like something like in below example
```
    @book = book
    book.rentals << self
```

Reply:
Hi, thank you but I'd like to point out that It does in fact already do this. that is the reason I am using `self.` instead of using the `@` symbol. because this will use the same functionality as the `book=` and `owner=` methods as opposed to having the same code that those functions are already doing twice.

Reply:
Yes, I see what you mean however, if you write it the way I propose you don't need to create the book and owner methods. for example, look at [this material](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) specifically how the `Visit` class is setup

Reply:
that's understandable Thank you for the feedback. My `book=` and `owner=` methods simply add the functionality for the book and owner to be changed at a later time and encapsulate this functionality in its own section. the constructor still works as it should.